openedx/features/job_board/views.py

```diff
@@ -1,9 +1,21 @@
+from django.views.generic.list import ListView
 from edxmako.shortcuts import render_to_response
 
+from .models import Job
 
-def list_jobs(request):
 
-    return render_to_response('features/job_board/job_list.html', {})
+class JobsListView(ListView):
+
+    model = Job
+    context_object_name = 'jobs_list'
```

Comment:
Please update **jobs_list** to **job_list** or **jobs**..

Reply:
`jobs_list` indicate that there are multiple jobs in this list instead of a single job in a list, as would `job_list` indicate. `jobs` does not communicate that this is a list.

Reply:
You are using plural with list, Even if you see django documentation example it use ArticleListView not ArticlesListView, and for context it is object_list not objects_list.

Secondly actually job_list doesn't contain list, but queryset. jobs will indicate that this variable has multiple jobs in it.

Reply:
Please have a look on it again